src/lib.rs

```diff
@@ -237,6 +237,14 @@ macro_rules! impl_rand {
                 }
             }
 
+            /// Create a new instance of the random number generator.
+            ///
+            /// This constructor is unsafe because it doesn't check that the CPU supports the
```

Comment:
```suggestion
            /// # Safety
            ///
            /// This constructor is unsafe because it doesn't check that the CPU supports the
```

Reply:
Done